gov/politeia/proposals.go

```diff
@@ -237,7 +236,7 @@ func (db *ProposalDB) CheckProposalsUpdates() error {
 }
 
 func (db *ProposalDB) lastSavedProposal() (lastP []*pitypes.ProposalInfo, err error) {
-	err = db.dbP.All(&lastP, storm.Limit(1), storm.Reverse())
+	err = db.dbP.Select().Limit(1).OrderBy("Timestamp").Reverse().Find(&lastP)
```

Comment:
@dmigwi At least in a sql statement, the order by has to come before limit.  Does this orm has a similar requirement?

I'm not seeing newer proposals on explorer.dcrdata.org/proposals, so something appears to be wrong despite it looking OK on alpha.

Reply:
The ORM has this example: 
```Golang
// Calls can also be chained
query = query.Limit(10).Skip(20).OrderBy("Age").Reverse()
```
https://github.com/asdine/storm

This is the newest proposal https://alpha.dcrdata.org/proposal/a-journey-to-the-future-of-work-telling-the-dcr-story-through-politeia

Reply:
Let me test the 4.1 PR on my machine.

Reply:
Thanks.  I'll scour the logs later.

Reply:
This test on my machine will not bear any fruits because my proposals db is upto date. From the https://explorer.dcrdata.org/proposals seems like upto six proposal could be missing.

Seems any relevant error or warning message on the logs?

Reply:
The main problem here is that dependencies used are linked to a specific version.
Since we've updated the Politeia package, you would need to update its dependency as linked in the `go.mod` file or just use a replace statement.

Reply:
Good catch.  I forget about this since we're using replace for everything on master.